github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/dsl/context/events/GitHubBranchCommitChecksDslContext.java

```diff
@@ -0,0 +1,29 @@
+package com.github.kostyasha.github.integration.branch.dsl.context.events;
+
+
+import com.github.kostyasha.github.integration.branch.events.impl.commitchecks.GitHubBranchCommitCheck;
+import com.github.kostyasha.github.integration.branch.events.impl.GitHubBranchCommitEvent;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javaposse.jobdsl.dsl.Context;
+import javaposse.jobdsl.plugin.ContextExtensionPoint;
+
+public class GitHubBranchCommitChecksDslContext implements Context {
+
+    private GitHubBranchCommitEvent event = new GitHubBranchCommitEvent();
+    private List<GitHubBranchCommitCheck> checks = new ArrayList<>();
+
+    public void commitMessagePattern(Runnable closure) {
+        GitHubBranchCommitMessageCheckDslContext commitContext = new GitHubBranchCommitMessageCheckDslContext();
+        ContextExtensionPoint.executeInContext(closure, commitContext);
+
+        checks.add(commitContext.getCheck());
+    }
+
+    public GitHubBranchCommitEvent getEvent() {
```

Comment:
@jgangemi so CommitEvent is Event because it implements *Events extension point. And checks are *Check extensionPoint. That what my initial idea about not mixing word Event in two different points (that also confuses camel completion in IDE)